Dockerfile

```diff
@@ -62,6 +62,9 @@ RUN case "$BUILDPLATFORM" in \
 ARG TARGETPLATFORM
 ENV TARGETPLATFORM=${TARGETPLATFORM:-linux/amd64}
 
+ARG GIT_COMMIT_ID
+ENV GIT_COMMIT_ID=${GIT_COMMIT_ID}
```

Comment:
Sorry for the necro-bump, but I noticed this change today, and as far as I can tell, this fucks up the Docker cache a bit... Please, mark me on any future changes to the Dockerfile. 🥲🙏

Reply:
You're right. Changing it invalidates all the next steps.

Any idea on how to improve upon this?

Reply:
Yeah, I'll open a PR with improvement (along with some other update/fixes) in a bit.

Reply:
Also, sorry if I sounded a bit harsh. This is not critical in any sense, it just pains me when I notice these sudden changes in the Dockerfile, after I painstakingly ordered every single instruction there. 😭